src/main/java/com/um/inventoryservice/BusinessLayer/DataSetupService.java

```diff
@@ -1,20 +1,16 @@
 package com.um.inventoryservice.BusinessLayer;
 
-import com.sun.jdi.ObjectReference;
-import com.um.inventoryservice.DataLayer.InventoryItemDTO;
+import com.um.inventoryservice.DataLayer.ClientDTO;
```

Comment:
10 Employee data samples will be added later, correct?

Reply:
yes i will add it in the getAll.